### DISCUSSION TITLE
Preferable way to configure default style

### DISCUSSION THREAD

**Discussion by taneba** [tamagui/tamagui#582] [Q&A]
Hi folks, I wan't to know what is the ideal way to configure default style to tamgui Components.

Let's say I'm using Text component everywhere, and set fontFamily="$body" every time.

```
<Text fontFamily="$body">text</Text>
```

## defaultProps

First thing I tried was to set defaultProps in tamagui.config.ts, but it didn't work:

```
  createTamagui({
    defaultProps: {
      Text: {
        fontFamily: "$body"
      }
    }
  })
```

## styled API
And I saw the docs saying "Prefer to use styled() for building your own".
But I just want to use Text as Text, not like MyText. And I want to use other tamgui's components without re-exporting and re-naming.

Does anyone have a good idea to achieve this?

**Comment by NandoMB** [comment]
I think it can help you https://github.com/tamagui/tamagui/discussions/788#discussioncomment-5338372
